### ISSUE TITLE
PostgreSqlTestcontainer fails to run if the image is not preexisting in Docker

### ISSUE THREAD

**Issue by JBBianchi** [testcontainers/testcontainers-dotnet#680]
**Describe the bug**
When starting a `TestcontainersBuilder<PostgreSqlTestcontainer>`, if the image `postgres:11.14` doesn exist, a parsing error will be thrown.
Notes: this behavior isn't consistent. It runs fine on Ubuntu 18.04 & Docker version 20.10.20 for instance.

**To Reproduce**
1. Remove the postgres image from Docker: `docker rmi -f $(docker images "postgres" -aq)` (in PowerShell)
2. Try to start a `TestcontainersBuilder<PostgreSqlTestcontainer>`:
```csharp
var container = new TestcontainersBuilder<PostgreSqlTestcontainer>()
    .WithName($"npgsql-{Guid.NewGuid():N}")
    .WithDatabase(new PostgreSqlTestcontainerConfiguration()
    {
        Database = "test",
        Username =  "test",
        Password =  "test"
    })
    .Build();
container.StartAsync().GetAwaiter().GetResult();
```
3. The exception is thrown:
```
  Message: 
Newtonsoft.Json.JsonReaderException : Error parsing undefined value. Path '', line 1, position 2.

  Stack Trace: 
JsonTextReader.MatchAndSetAsync(String value, JsonToken newToken, Object tokenValue, CancellationToken cancellationToken)
JsonTextReader.ParseValueAsync(CancellationToken cancellationToken)
StreamUtil.MonitorStreamForMessagesAsync[T](Task`1 streamTask, DockerClient client, CancellationToken cancellationToken, IProgress`1 progress)
StreamUtil.MonitorResponseForMessagesAsync[T](Task`1 responseTask, DockerClient client, CancellationToken cancel, IProgress`1 progress)
DockerImageOperations.CreateAsync(IDockerImage image, IDockerRegistryAuthenticationConfiguration dockerRegistryAuthConfig, CancellationToken ct)
TestcontainersClient.RunAsync(ITestcontainersConfiguration configuration, CancellationToken ct)
TestcontainersContainer.Create(CancellationToken ct)
TestcontainersContainer.StartAsync(CancellationToken ct)
...
``` 

**Expected behavior**
The container to run.

**Env**
Win 10 x64
Docker version 20.10.20, build 9fdeb9c
Docker Desktop 4.13.1

**Comment by HofmeisterAn** [comment]
Which version of Testcontainers for .NET do you use? There was #634 which is fixed in `2.2.0`. Someone else used a wrong image name: https://github.com/testcontainers/testcontainers-dotnet/issues/672#issuecomment-1311865246. Is the Linux engine enabled? Can you add the output of `docker info`?

**Comment by JBBianchi** [comment]
@HofmeisterAn thanks for your answer, I should have checked the version indeed. The project was in `2.1.0`, updating it to `2.2.0` solved the issue. Thanks again!
